### ISSUE TITLE
Problem with UI locking up on opening a F# file/project

### ISSUE THREAD

**Issue by 7sharp9** [mono/monodevelop#5545]
REPO: 

*   clone https://github.com/fsprojects/FSharp.TypeProviders.SDK
*   open the FSharp.TypeProviders.SDK solution
*   open providedtypes.fs

UI hangs with the lollipop of doom for around a 60-90 seconds.

> VS bug [#656654](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/656654)

**Comment by 7sharp9** [comment]
OSX, latest from alpha channel, also occurs on stable and beta
```
Visual Studio Community 2017 for Mac (Preview)
Version 7.6 Preview (7.6 build 2086)
Installation UUID: fa9b85bb-1df2-4bcc-83ad-cbf1ff9a59f9
Runtime:
	Mono 5.12.0.298 (2018-02/f795aa5d2db) (64-bit)
	GTK+ 2.24.23 (Raleigh theme)
	Xamarin.Mac 4.4.1.178 (master / eeaeb7e6)

	Package version: 512000298
```

**Comment by nosami** [comment]
@monojenkins backport release-7.6

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-20] Comment by **xamarinc**



Fixed in version 8.6.0.802 (master)



Pull Request #5550 merged by: Jason Imison  
 Author: mono  
 Commit: a3b5ee10760a5fb83ce6defc4cd924167decf061 (xamarin/monodevelop)
